Update Client SDK example with multi-org support (#565)
hyperledger-labs/fabric-private-chaincode: integration/test-network/README.md

In order to initiate the FPC Chaincode enclave and register it with the FPC Enclave Registry, run the app with the `-withLifecycleInitEnclave` flag.

```bash
cd $FPC_PATH/client_sdk/go/sample
CC_ID=echo ORG_NAME=Org1 go run . -withLifecycleInitEnclave
```
Note that we execute the go app as `Org1`, thereby creating and registering the FPC Chaincode enclave at `peer0.org1.example.com`. Alternativley, we could run this as `Org2` to initiate the enclave at `peer0.org2.example.com`.

Afterwards you _must_ run the application without the `withLifecycleInitEnclave` flag and you can play with multiple organizations.
```bash
cd $FPC_PATH/client_sdk/go/sample
CC_ID=echo ORG_NAME=Org1 go run .
CC_ID=echo ORG_NAME=Org2 go run .
```

## Shutdown network
```bash
make -C $FPC_PATH/integration/test-network ercc-ecc-stop
cd $FPC_PATH/integration/test-network/fabric-samples/test-network
./network.sh down
```

## Debugging

For diagnostics, you can run the following to see logs for `peer0.org1.example.com`.
```bash
docker logs -f peer0.org1.example.com
docker logs -f ercc.peer0.org1.example.com
docker logs -f ecc.peer0.org1.example.com
```

> To interact interactively with the peer using the `peer CLI`, run the following

Author: ```suggestion
To interact with the peer using the `peer CLI`, run the following
```

```bash
cd ${FPC_PATH}/client_sdk/go/sample
CC_ID=echo ORG_NAME=Org1 go run . -withLifecycleInitEnclave
```

> Note that we execute the go app as `Org1` and thereby issue to create and registery the FPC Chaincode enclave at `peer0.org1.example.com`.

Reviewer: ```suggestion
Note that we execute the go app as `Org1`, thereby creating and registering the FPC Chaincode enclave at `peer0.org1.example.com`.
```

> Note that we execute the go app as `Org1` and thereby issue to create and registery the FPC Chaincode enclave at `peer0.org1.example.com`.

Reviewer: Should we add that, alternatively, we could do the same at Org2?

> Afterwards you can run the application without the `withLifecycleInitEnclave` flag and play with multiple organizations.

Reviewer: Actually, if by chance somebody runs "with" initEnclave, it will fail. So, it is mandatory to use it once and only once.
```suggestion
Afterwards you _must_ run the application without the `withLifecycleInitEnclave` flag and you can play with multiple organizations.
```
